src/overlays/actors/ovl_Obj_Grass/z_obj_grass.c

```diff
@@ -7,6 +7,7 @@
 #include "z_obj_grass.h"
 #include "overlays/actors/ovl_Obj_Grass_Carry/z_obj_grass_carry.h"
 #include "objects/gameplay_keep/gameplay_keep.h"
+#include "prevent_bss_reordering.h"
```

Comment:
I think including this header _after_ the others doesn't do anything, could be wrong tho. Could you check here?

Reply:
Nope, definitely required. And I don't see why it wouldn't do anything.